ui/app/mixins/policy-edit-controller.js

```diff
@@ -28,13 +27,19 @@ export default Ember.Mixin.create({
       let flash = this.get('flashMessages');
       let policyType = model.get('policyType');
       let name = model.get('name');
-      model.save().then(m => {
-        flash.success(`${policyType.toUpperCase()} policy "${name}" was successfully saved.`);
-        if (this.get('wizard.featureState') === 'create') {
-          this.get('wizard').transitionFeatureMachine('create', 'CONTINUE', policyType);
-        }
-        return this.transitionToRoute('vault.cluster.policy.show', m.get('policyType'), m.get('name'));
-      });
+      model
+        .save()
+        .then(m => {
+          flash.success(`${policyType.toUpperCase()} policy "${name}" was successfully saved.`);
+          if (this.get('wizard.featureState') === 'create') {
+            this.get('wizard').transitionFeatureMachine('create', 'CONTINUE', policyType);
+          }
+          return this.transitionToRoute('vault.cluster.policy.show', m.get('policyType'), m.get('name'));
+        })
+        .catch(() => {
+          // do nothing here...
+          // message-error component will show errors
+        });
```

Comment:
They're added because if we don't catch, the tests blow up (test bail on uncaught promise rejection) - I don't remember why this wasn't the case previously - could've just been an old enough ember, but also with the module change, stubbing out Ember.test.adapter to get around this behavior no longer worked.